types/node/buffer.d.ts

```diff
@@ -169,8 +169,8 @@ declare module 'buffer' {
     import { Blob as NodeBlob } from 'buffer';
     // This conditional type will be the existing global Blob in a browser, or
     // the copy below in a Node environment.
-    type __Blob = typeof globalThis extends { onmessage: any, Blob: infer T }
-        ? T : NodeBlob;
+    type __Blob = typeof globalThis extends { onmessage: any }
```

Comment:
I think we should at least be checking whether `Blob` is present, no?

```suggestion
    type __Blob = typeof globalThis extends { onmessage: any, Blob: any }
```

... and the same for the other interfaces.

Reply:
If the need is not clear, fixing the bug should be a priority.